onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -1301,7 +1301,7 @@ including arg name, arg type (contains both type and shape).)pbdoc")
           },
           "node shape (assuming the node holds a tensor)");
 
-  py::class_<SessionObjectInitializer>(m, "SessionObjectInitializer");
+  py::class_<SessionObjectInitializer> sessionObjectInitializer(m, "SessionObjectInitializer");
```

Comment:
why only change this one? the next line (and others) looks similar

Reply:
The next two at least they use the obj for doing something. There are dots after the constructor. This one doesn't. So VC++ only complains this one.